app/src/main/java/in/testpress/testpress/models/InstituteSettings.java

```diff
@@ -467,6 +468,16 @@ public long serverTime() {
         }
         return 0L;
     }
+
+    public VerificationMethod getVerificationType(InstituteSettings instituteSettings) {
```

Comment:
why is this static method ?

Reply:
This is not a static method

Reply:
Then why usage is not directly on institute object ?